helpers/index.js

```diff
@@ -1,8 +1,13 @@
-export function sumofdigits(num) {
-if (num < 10) {
-  return num;
-}else{
-  return (num % 10) + sumofdigits (Math.floor(num/10));
-}
- 
-}
+export function sumArray(arr) {
+  let total = 0;
+
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
Create another function that helps you go through all of the elements (don't use a loop)